base/loading.jl

```diff
@@ -66,40 +67,37 @@ function _require_from_serialized(node::Int, path_to_try::ByteString, toplevel_l
         else
             content = remotecall_fetch(node, open, readbytes, path_to_try)
         end
-        if _include_from_serialized(content)
+        restored = _include_from_serialized(content)
+        if restored != nothing
```

Comment:
@yuyichao, I just started to do this, and then I noticed that there are about 100 ` == nothing` and ` != nothing` checks scattered throughout Base, and this is common in packages as well.

I think the right thing to do is just to define an optimized `==` method for `==(::Any, ::Void)`.